### PR TITLE
WINDUPRULE-400: eap 7.3 move to mp rest client 1.3

### PR DESCRIPTION
**Pull request by emmartins**
Issue: https://issues.redhat.com/browse/WINDUPRULE-400

### CODE REVIEW

**Comment by emmartins** [comment]
review only, not ready for merge due to <link /> FIXME

File: rules-reviewed/eap7/eap72/move-to-microprofile-rest-client-1.3.rhamt.xml [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+        <!-- https://issues.jboss.org/browse/WINDUPRULE-400 -->
+        <rule id="move-to-microprofile-rest-client-1.3-00001">
+            <when>
+                <javaclass references="org.jboss.resteasy.client.microprofile.MicroprofileClientBuilderResolver" >
+                    <location>IMPORT</location>
+                </javaclass>
+            </when>
+            <perform>
+                <hint title="Move to Microprofile REST Client 1.3" effort="1" category-id="potential">
+                    <message>The application imports class 'org.jboss.resteasy.client.microprofile.MicroprofileClientBuilderResolver', and should be changed to use instead Microprofile REST Client 1.3.</message>
```
Comment (mrizzi) [inline rules-reviewed/eap7/eap72/move-to-microprofile-rest-client-1.3.rhamt.xml]: I think it would help to mention to use `RestClientBuilder` instead.

Comment (emmartins) [reply rules-reviewed/eap7/eap72/move-to-microprofile-rest-client-1.3.rhamt.xml]: changed, is it ok now?

File: rules-reviewed/eap7/eap72/move-to-microprofile-rest-client-1.3.rhamt.xml [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+        <!-- https://issues.jboss.org/browse/WINDUPRULE-400 -->
+        <rule id="move-to-microprofile-rest-client-1.3-00001">
+            <when>
+                <javaclass references="org.jboss.resteasy.client.microprofile.MicroprofileClientBuilderResolver" >
+                    <location>IMPORT</location>
+                </javaclass>
+            </when>
+            <perform>
+                <hint title="Move to Microprofile REST Client 1.3" effort="1" category-id="potential">
+                    <message>The application imports class 'org.jboss.resteasy.client.microprofile.MicroprofileClientBuilderResolver', and should be changed to use instead 'org.eclipse.microprofile.rest.client.RestClientBuilder', from Microprofile REST Client 1.3.</message>
```
Comment (mrizzi) [inline rules-reviewed/eap7/eap72/move-to-microprofile-rest-client-1.3.rhamt.xml]:
```suggestion
                    <message>The application imports class `org.jboss.resteasy.client.microprofile.MicroprofileClientBuilderResolver`, and should be changed to use instead `org.eclipse.microprofile.rest.client.RestClientBuilder`, from Microprofile REST Client 1.3.</message>
```

File: rules-reviewed/eap7/eap72/move-to-microprofile-rest-client-1.3.rhamt.xml (hunk 1 above)
Comment (mrizzi) [reply rules-reviewed/eap7/eap72/move-to-microprofile-rest-client-1.3.rhamt.xml]: perfect :+1:

File: rules-reviewed/eap7/eap72/move-to-microprofile-rest-client-1.3.rhamt.xml (hunk 2 above)
Comment (emmartins) [reply rules-reviewed/eap7/eap72/move-to-microprofile-rest-client-1.3.rhamt.xml]: done

**Comment by mrizzi** [comment]
retest

**Pull request merged**